### ISSUE TITLE
Keras Functional API

### ISSUE THREAD

**Issue by Traitor197** [SciSharp/Keras.NET#20]
Hello. How can I implement the Functional API instead of Sequential models?
I cant figure out how it works with Keras.NET and cant find something.
I tried this appoach:
`var input = new Input(new Keras.Shape(64, 12));
var a = new CuDNNLSTM(32);
a.Set(input);
var output = new Dense(1, activation: "sigmoid");
output.Set(a);
var model = new Keras.Models.Model(new Input[] { input }, new BaseLayer[] { output });`

I get this error in the last line:
Python.Runtime.PythonException: "ValueError : Input tensors to a Model must come from `keras.layers.Input`. Received: Keras.Layers.Input[] (missing previous layer metadata)."

However I must give a BaseLayer[] Array as argument.
Are layers even connected by the Set() method?

**Comment by deepakkumar1984** [comment]
I found the issue and working on it. I have two more related issues, will update the nuget with the fixes.

**Comment by deepakkumar1984** [comment]
Resolved with latest checkin

**Comment by GeorgeS2019** [comment]
The fix is this [PR](https://github.com/SciSharp/Keras.NET/pull/60/files)
